rule/return-argument-limit.go

```diff
@@ -0,0 +1,65 @@
+package rule
+
+import (
+	"fmt"
+	"go/ast"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// ReturnArgumentsLimitRule lints given else constructs.
+type ReturnArgumentsLimitRule struct{}
+
+// Apply applies the rule to given file.
+func (r *ReturnArgumentsLimitRule) Apply(file *lint.File, arguments lint.Arguments) []lint.Failure {
+	if len(arguments) != 1 {
+		panic(`invalid configuration for "return-argument-limit"`)
+	}
+
+	total, ok := arguments[0].(int64) // Alt. non panicking version
```

Comment:
`total` could be renamed to `max`
(the same for the field `total` of the `lintReturnArgumentsNum` struct)